### PR TITLE
#51 - change package sign

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #51 
I've changed `Package` interface signature to make it possible to update/add info about several `deb` packages in one update. 
Also, removed `Release` index update from `Package.Asto` implementation, it does not correspond to `Debian` interface workflow.

### CODE REVIEW

**Comment by olenagerasimova** [comment]
@artipie/contributors could someone please review this PR?

File: src/main/java/com/artipie/debian/metadata/Package.java [hunk 1]
```diff
@@ -47,11 +48,11 @@
 
     /**
      * Adds item to the packages index.
-     * @param item Index item to add
+     * @param items Index items to add
      * @param index Package index key
      * @return Completion action
      */
-    CompletionStage<Void> add(String item, Key index);
+    CompletionStage<Void> add(List<String> items, Key index);
```
Comment (genryxy) [inline src/main/java/com/artipie/debian/metadata/Package.java:55]: @olenagerasimova probably to use here `Iterable<String>` instead of `List<String>`

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, one suggestion

**Comment by olenagerasimova** [comment]
@genryxy thanks, corrected

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks, looks good to me

**Pull request merged**